### PR TITLE
Add UntitledUI Icons set

### PR DESCRIPTION
**Pull request by mckenziearts**
Add Untitled UI icons from this [PR](https://github.com/blade-ui-kit/blade-icons/pull/216) to add the [blade-untitledui-icons](https://github.com/mckenziearts/blade-untitledui-icons) package to the list of packages so they can be discovered.

### CODE REVIEW

**Comment by driesvints** [comment]
Thanks! Seems tests still fail here.

File: app/Models/IconSet.php [hunk 1]
```diff
@@ -585,6 +585,14 @@ final class IconSet extends Model
             'ignore_rule' => null,
             'outline_rule' => null,
         ],
+        [
+            'id' => 73,
```
Comment (nuernbergerA) [inline app/Models/IconSet.php:589]: this should be 74

**Comment by driesvints** [comment]
There still seems to be something wrong here..

**Comment by mckenziearts** [comment]
Please Re-run the jobs, I published a new release of the original package and I updated the icon Set. That's why there was this error.

**Comment by nuernbergerA** [comment]
@mckenziearts you need to update and commit the composer.lock

**Comment by driesvints** [comment]
Thanks all!

**Pull request merged**